src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/steps/Step3.tsx

```diff
@@ -31,42 +36,42 @@ export function Step3({ nativeTokenSymbol, tokenLabel, order, refund, cancelatio
       return {
         label: 'Receive ' + tokenLabel,
         state: 'pending',
-        icon: Flag,
+        icon: Finish,
       }
     }
     if (isIndexing) {
       return {
         label: 'Receive ' + tokenLabel,
         state: 'not-started',
-        icon: Flag,
+        icon: Finish,
       }
     }
     if (isFilled) {
       return {
-        label: 'Received ' + tokenLabel,
+        label: 'Received ' + tokenLabel + '!',
```

Comment:
Are you sure we want this exclamation mark. I feel, it messes up a bit with the token name (just a nit)

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/2352112/204856031-16579427-95c6-4775-86d2-b14f78d4d4a9.png">

Reply:
We can remove it

Reply:
removed it